capabilities/connect.capabilities.yml

```diff
@@ -139,6 +139,170 @@ properties:
                 type: string
                 examples:
                 - Ethernet1
+  mclag-capabilities:
+    description: Capabilities of the MC-LAG (Multi-Chassis Link Aggregation) feature in the switch
+    type: object
+    properties:
+      max-mclag-groups:
+        description: Defines the maximum number of MC-LAG groups that can be configured on the switch.
+        type: integer
+      max-ports-per-mclag-group:
+        description: Specifies the maximum number of physical ports that can be part of a single MC-LAG group.
+        type: integer
+      max-vlans-per-mclag-group:
+        description: Indicates the maximum number of VLANs that can be supported within a single MC-LAG group.
+        type: integer
+      peer-link-bandwidth:
+        description: Defines the required minimum bandwidth for the peer link that connects two MC-LAG switches.
+        type: string
+        enum:
+          - 1G
+          - 10G
+          - 40G
+          - 100G
+      dual-active-detection:
```

Comment:
For things like this, since the configuration is a boolean, would it not make more sense to have an attribute like dual-active-detection be a boolean, as it is enabled, or not, that is all you can do with it configuration wise.

Reply:
I prefer to leave it this way for now. I do see some switches do not support the ICCP yet. In those cases, it'd be good to have it said in the capabilities. I will leave this open for now to discuss in next Sprint. OK with you?